ansible/bashrc_ldcoolp_alias_setup.yml

```
@@ -16,6 +16,7 @@
         ldcoolp_config='$ldcoolp_root/ldcoolp/config/arizona.ini'
         alias get_pending="$ldcoolp_root/ldcoolp/scripts/get_curation_list --config $ldcoolp_config"
         alias get_data="$ldcoolp_root/ldcoolp/scripts/prereq_script --config $ldcoolp_config --article_id "
+        alias get_metadata="$ldcoolp_root/ldcoolp/scripts/prereq_script --config $ldcoolp_config --article_id --metadata_only"
```

```suggestion
        alias get_metadata="$ldcoolp_root/ldcoolp/scripts/prereq_script --config $ldcoolp_config --metadata_only --article_id "
```

Actually the order or flags needs to be flipped since the article_id is passed in to the alias at the end.